### PR TITLE
Release note documents for v10.0.3

### PR DESCRIPTION
**Pull request by frouioui**
## Description
This pull request updates the release note documents following the release of v10.0.3.

### CODE REVIEW

File: doc/releasenotes/10_0_3_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#### Build/CI
+CI: ubuntu-latest now has MySQL 8.0.26, let us override it with latest 8.0.x #9375
+### Internal Cleanup
+#### Java
+* build(deps): bump log4j-api from 2.13.3 to 2.15.0 in /java #9363
+
+
+The release includes 170 commits (excluding merges)
+
+Thanks to all our contributors: @GuptaManan100, @aquarapid, @askdba, @deepthi, @dweitzman, @dyv, @frouioui, @guidoiaquinti, @harshit-gangal, @jmoldow, @rohit-nayak-ps, @shlomi-noach, @sougou, @systay
```
Comment (deepthi) [inline doc/releasenotes/10_0_3_release_notes.md:19]: This doesn't seem accurate, there are only 2 PRs.

File: doc/releasenotes/10_0_3_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Release of Vitess v10.0.3
+## Announcement
+
+This patch is providing an update regarding the Apache Log4j security vulnerability (CVE-2021-44228) (#9363), along with a few bug fixes.
```
Comment (deepthi) [inline doc/releasenotes/10_0_3_release_notes.md:4]: I think we can leave out the "along with a few bug fixes". There aren't any.

File: doc/releasenotes/10_0_3_summary.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Major Changes
+
+This patch is providing an update regarding the Apache Log4j security vulnerability (CVE-2021-44228) (#9363), along with a few bug fixes.
```
Comment (deepthi) [inline doc/releasenotes/10_0_3_summary.md:3]:
```suggestion
This patch is providing an update regarding the Apache Log4j security vulnerability (CVE-2021-44228) (#9363).
```

File: doc/releasenotes/10_0_3_release_notes.md (hunk 1 above)
Comment (frouioui) [reply doc/releasenotes/10_0_3_release_notes.md]: Some old pull requests got included in the release notes somehow. I've fixed it by listing 2 contributors and 5 commits.

**Review by askdba — APPROVED** [review]
LGTM

**Pull request merged**